src/main/scala/machines/Internal.scala

```diff
@@ -2,5 +2,52 @@ package machines
 
 import regex._
 import dfa._
+import machines._
 
-// TODO: Add your code below
+// implicit conversion from char to regular language
+given Conversion[Char, RegularLanguage] = Character(_)
+
+
+// implicit conversion from string to regular language
+given Conversion[String, RegularLanguage] with
+    def apply(s: String): RegularLanguage = {
+        if (s.isEmpty)
+            Epsilon
+        else 
+            Concat(Character(s.head), apply(s.tail))
```

Comment:
Using a recursive function like this is one good way to create a string conversion, but it's interesting that scala allows you to define a function in this way!